locales-pending/landing-all.ftl

```diff
@@ -9,3 +9,17 @@ landing-all-hero-emailform-submit-label = Get free scan
 
 # This is a label underneath a big number "14" - it's an image that demos Monitor.
 landing-all-hero-image-chart-label = exposures
+
+# Value Proposition 
+
+landing-all-value-prop-fix-exposures = We'll help you fix your exposures
+landing-all-value-prop-fix-exposures-description = Our mission is to put control of your personal data back in your hands. We’ll help you resolve data breaches and keep your info private – and we’ll respect your privacy in the process.
+landing-all-value-prop-info-at-risk = What info could be at risk?
+landing-all-value-prop-info-at-risk-description = Data leaks are unfortunately part of our digital lives. Your passwords, contact details, financial information, and other personal info can be exposed, putting you at risk of identity theft.
+landing-all-value-prop-scanning-for-exposures = Scanning for exposures...
+landing-all-value-prop-get-started = Get started
+landing-all-value-prop-enter-info = Enter info
+landing-all-value-prop-find-exposures = Find exposures
+landing-all-value-prop-exposure-example-leaked-pw = Leaked Passwords
+landing-all-value-prop-exposure-example-fix-it = Let's fix it
+landing-all-value-prop-exposure-example = Your glasses.com password was exposed
```

Comment:
Using a real website in an example sounds like a terrible idea?

Also, please leave an empty line at the end of the file.